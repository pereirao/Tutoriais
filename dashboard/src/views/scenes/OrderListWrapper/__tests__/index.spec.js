import React from 'react';
// NOTE: named import is not connected component
import { OrderListWrapper } from '../index';

it('renders correctly', () => {
  expect(shallow(<OrderListWrapper location={{pathname: '/'}} />)).toMatchSnapshot();
});
