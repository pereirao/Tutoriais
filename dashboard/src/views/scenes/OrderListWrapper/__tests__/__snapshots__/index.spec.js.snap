// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sceneWrapper"
>
  <Connect(NotificationFeed) />
  <div
    className="orderListsWrapper"
  >
    <div
      className="tabWrapper"
    >
      <div
        className="tab"
        onClick={[Function]}
        role="link"
        tabIndex={0}
        title="Orders that are relevant at present"
      >
        Active Orders
      </div>
      <div
        className="tab"
        onClick={[Function]}
        role="link"
        tabIndex={1}
        title="Orders that have been finished or canceled"
      >
        Completed Orders
      </div>
      <div
        className="tab"
        onClick={[Function]}
        role="link"
        tabIndex={2}
        title="Orders that are scheduled for later days"
      >
        Future Orders
      </div>
      <div
        className="tab"
        onClick={[Function]}
        role="link"
        tabIndex={3}
        title="Orders from all time"
      >
        All Orders
      </div>
      <div
        className="tabBuffer"
      />
    </div>
    <div
      className="tabContentWrapper"
    >
      <Switch>
        <Route
          component={[Function]}
          exact={true}
          path="/orders/active"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/orders/completed"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/orders/future"
        />
        <Route
          component={[Function]}
          exact={true}
          path="/orders/all"
        />
      </Switch>
    </div>
  </div>
</div>
`;
